src/ofApp.h

```diff
@@ -31,6 +31,6 @@ class ofDisplayApp : public ofBaseApp {
 
 	Covid19 covid19;
 
-	void loadCovidCsv();
+  void loadCovidCsv();
```

Comment:
Formatting thrash? (I swear, that's not the only thing I pay attention to in a codebase.)